### PR TITLE
Adding template support in qbol operator

### PR DESCRIPTION
**Pull request by msumit**

### CODE REVIEW

File: airflow/contrib/operators/qubole_operator.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 from airflow.utils import apply_defaults
 from airflow.contrib.hooks import QuboleHook
 
-
```
Comment (mistercrunch) [inline airflow/contrib/operators/qubole_operator.py:5]: PEP8 says 2 lines before class definitions

**Comment by mistercrunch** [comment]
I feel like all these attributes aren't guaranteed to be present and that the framework will choke trying to template missing attributes. The contract of what one should provide is unclear.

If they are required, they should be explicitly defined in the constructor header (with defaults), and stamped into `self` in **init**.py.

**Comment by mistercrunch** [comment]
I read the way things are organized in the hook and operator and it's a bit messy the way kwargs is passed around. I saw how you overrode the **get_attriubte** is a bit hacky too. The operator doesn't have a good doc string to compensate for this behavior... Consider making a clearer interface to the operator, or documenting its expected behavior in the doc string.

**Comment by msumit** [comment]
Hi Maxime, 

I do agree with your concerns. Actually we didn't wanted to add dozens of new operators like `QuboleHiveOperator`, `QuboleHadoopOperator` etc, so we'd to do hacks like that. If we had individual command operators like I said above, then we could adhere to simple design like the other operators have. 

So either I can try to write down the behaviour in more better way, or you can suggest me some alternative approach for supporting templates?

Thanks,
Sumit

**Comment by mistercrunch** [comment]
:+1:

**Pull request merged**